app/views/shared/_header.html.erb

```diff
@@ -4,6 +4,12 @@
     <span class="menu-expando tws-logo-type" onclick='toggleHeader()'></span>
     <nav class='nav-list'>
       <ul class='nav-links'>
+        <% if current_user && current_user.host? %>
+          <li class="nav-link-item">
+            <%= link_to current_user.nickname, host_city_path(current_user.home_city, current_user) %>
```

Comment:
`host_city_path` doesn't really make sense to me. We should change the name of this path to something that is more self explanatory. Outside the scope of this PR though.